# Scan Results for flamman.se

*Scan ID: `adcfb440-9c49-44a3-b7c8-ce355d6c9ff2`*
*Scan started: 2025-04-23 13:18:45*

## Initializing Scan

Please wait while we analyze the target. This page will update automatically.

## Scanning in Progress...

The scan tools (Nmap, Nikto, ZAP, etc.) are now running. Results will be appended below as they become available.


## Error During Scan Process

A critical error occurred during the scan execution:
```
Traceback (most recent call last):
  File "/home/optibot/Degree/frontend.py", line 110, in run_scan_process
    scan.run_scan(domain=domain, scan_id=scan_id)
TypeError: run_scan() got an unexpected keyword argument 'scan_id'

```
